pymtl3/passes/autotick/OpenLoopCLPass.py

```diff
@@ -31,16 +32,19 @@
 
 class OpenLoopCLPass( BasePass ):
   def __init__( self, print_line_trace=True ):
-    self.line_trace_on = print_line_trace
+    self.print_line_trace = print_line_trace
 
   def __call__( self, top ):
     if not hasattr( top._dag, "all_constraints" ):
       raise PassOrderError( "all_constraints" )
 
     top._sched = PassMetadata()
+    top._sim   = PassMetadata()
```

Comment:
I think before the release we are gonna use the metadata APIs and deprecate `PassMetadata`?

Reply:
You are correct. I'm not 100% sure about how to use metadata APIs now and those code are developed before the metadata APIs are merged. Can you point me to an example?